packages/platforms/browser/lib/persistence.ts

```diff
@@ -28,6 +28,53 @@ function makeBrowserPersistence (window: WindowWithLocalStorage): Persistence {
   return new InMemoryPersistence()
 }
 
+// NOTE: this should be kept in sync with the notifier
+// https://github.com/bugsnag/bugsnag-js/blob/next/packages/plugin-browser-device/device.js
+function isDeviceId (raw: string): boolean {
+  // make sure the persisted value looks like a valid cuid
+  return /^c[a-z0-9]{20,32}$/.test(raw)
+}
```

Comment:
could we put this in our own fork of cuid, and import it in both packages?

Reply:
Yeah, good idea. Also the `.d.ts` file

https://smartbear.atlassian.net/browse/PLAT-10305